core-bundle/contao/library/Contao/TemplateTrait.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao;
+
+use Contao\CoreBundle\Routing\ResponseContext\Csp\CspHandler;
+use Contao\CoreBundle\Routing\ResponseContext\JsonLd\JsonLdManager;
+use Contao\CoreBundle\String\HtmlAttributes;
+use Contao\Image\ImageInterface;
+use Contao\Image\PictureConfiguration;
+use Spatie\SchemaOrg\Graph;
+
+/**
+ * Provides template methods.
```

Comment:
Changes to the legacy code should IMHO be internal if possible. 🙂

```suggestion
 * Provides template methods.
 *
 * @internal
```